packages/formdata-event/ts_src/formdata_listener_added.ts

```diff
@@ -167,19 +188,22 @@ export const wrapSubmitListener = (listener: EventListenerOrEventListenerObject)
     // event.
     if (getEventPropagationImmediatelyStopped(e) && targetIsAForm) {
       maybeDispatchFormdataForEvent(e);
-    // If the event's propagation was stopped (potentially before being seen by
-    // the capturing 'submit' listener) and this is the _last_ callback for its
-    // event phase, dispatch a 'formdata' event.
+      // If the event's propagation was stopped (potentially before being seen by
+      // the capturing 'submit' listener) and this is the _last_ callback for its
+      // event phase, dispatch a 'formdata' event.
     } else if (getEventPropagationStopped(e) && targetIsAForm) {
       const submitListeners = targetToSubmitListeners.get(this)!;
       const {lastCapturingCallback, lastBubblingCallback} = submitListeners;
 
-      if (wrapper === lastCapturingCallback || wrapper === lastBubblingCallback) {
+      if (
+        wrapper === lastCapturingCallback ||
+        wrapper === lastBubblingCallback
+      ) {
         maybeDispatchFormdataForEvent(e);
       }
-    // If this listener is the _last_ bubbling 'submit' event listener attached
-    // to the shallow root of the event's target (possibly the one added by the
-    // capturing listener), dispatch a 'formdata' event.
+      // If this listener is the _last_ bubbling 'submit' event listener attached
+      // to the shallow root of the event's target (possibly the one added by the
+      // capturing listener), dispatch a 'formdata' event.
     } else if (submitEventToListenerInfo.has(e)) {
```

Comment:
```suggestion
    }
    // If this listener is the _last_ bubbling 'submit' event listener attached
    // to the shallow root of the event's target (possibly the one added by the
    // capturing listener), dispatch a 'formdata' event.
    else if (submitEventToListenerInfo.has(e)) {
```

(same comment as above)